lib/xclarity_client/client.rb

```diff
@@ -7,19 +7,19 @@ def initialize(connection)
     end
 
     def discover_nodes(opts = {})
-      NodeManagement.new(@connection).population opts
+      NodeManagement.new(@connection).fetch_all(opts)
```

Comment:
What do you think @douglasgabriel about rename @connection to @data or @config etc, because always it is XClarityClient::Configuration object, make more sense for you?